openstack/ecs/v1/block_devices/requests.go

```diff
@@ -2,9 +2,108 @@ package block_devices
 
 import (
 	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/ecs/v1/jobs"
 )
 
-func Get(c *golangsdk.ServiceClient, server_id string, volume_id string) (r GetResult) {
-	_, r.Err = c.Get(getURL(c, server_id, volume_id), &r.Body, nil)
-	return
+var requestOpts = golangsdk.RequestOpts{
+	MoreHeaders: map[string]string{"Content-Type": "application/json", "X-Language": "en-us"},
+}
+
+// Get is a method to obtain the detail of volume attachment for specified cloud server.
+func Get(c *golangsdk.ServiceClient, serverId string, volumeId string) (*Attachment, error) {
```

Comment:
why change the return type?

Reply:
Changes have been reworked to be compatible with legacy code